lessons/213/k8s/deploy/go-app/1-deployment.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   name: go-app
   namespace: default
 spec:
-  replicas: 2
+  replicas: 1
```

Comment:
thats not very fair to bun to have 1 dedicated pod vs 2 smaller pods in bun

https://github.com/antonputra/tutorials/blob/main/lessons/213/k8s/deploy/bun-app/1-deployment.yaml

Reply:
I disagree, requiring 2 pods for bun is a limitation, running 2 pods for bon is arguably unfair but reasonable.

Reply:
This is due to "hidden" knowledge that Bun is single core. Okay

Reply:
I'll try bun workers next time but I'll definitely keep the same number of pods - https://bun.sh/docs/api/workers